src/components/calcite-combobox-item/calcite-combobox-item.tsx

```diff
@@ -37,7 +39,7 @@ export class CalciteComboboxItem {
   @Prop() active = false;
 
   /** Parent and grandparent combobox items, this is set internally for use from combobox */
-  @Prop({ mutable: true }) anscestors: HTMLCalciteComboboxItemElement[];
+  @Prop({ mutable: true }) anscestors: ComboboxAncestorElement[];
```

Comment:
Typo: `ancestors`

Reply:
eek. this would be a breaking change not related to this PR :\

Should I still make it?

Reply:
SPELL CHECK :)

Reply:
this prop is really only expected to be set internally by us. To the end dev, they will most likely not set this directly. I think it would be fine to update.